content/docs/reference/microcontrollers/nano-33-ble.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Arduino Nano33 BLE"
+weight: 3
+---
+
```

Comment:
Probably also add info about “Sense” variant that has same pinout just more sensors built-in on the same board, mainly on I2C bus.